README.md

```diff
@@ -1,12 +1,11 @@
-open-orchestra-media-bundle
+open-orchestra-workflow-bundle
```

Comment:
No, I think that we should let a bundle for this specific task, so when you integrate it, you will only have the part you need and no other part